hybridCarboniteMetadataService/src/main/scala/cromwell/services/metadata/hybridcarbonite/CarbonitedMetadataThawingActor.scala

```diff
@@ -78,6 +91,26 @@ final class CarbonitedMetadataThawingActor(carboniterConfig: HybridCarboniteConf
       log.error(s"Programmer Error: Unexpected message to ${getClass.getSimpleName} ${self.path.name}: $other")
       stay()
   }
+
+  private def rootWorkflowIdAndJsonToGcsResponseAndLabelsRequest(action: BuildWorkflowMetadataJsonAction,
+                                                                 rootWorkflowIdAndJson: (String, Json)): (GcsMetadataResponse, Option[GetRootAndSubworkflowLabels]) = {
+    rootWorkflowIdAndJson match {
+      case (rootWorkflowId, rootWorkflowJson) =>
```

Comment:
since this is a one-case match, you might want to express it like:
```
val rootWorkflowId, rootWorkflowJson = rootWorkflowIdAndJson
``` 

Alternatively... could you make the method signature be like this?
```
  private def rootWorkflowIdAndJsonToGcsResponseAndLabelsRequest(action: BuildWorkflowMetadataJsonAction,
                                                                 rootWorkflowId: String
                                                                 rootWorkflowJson: Json)
```